custom.css

```
@@ -25,17 +25,36 @@
   margin-top: -0.25rem;
 }
 
-.GrowImage {
-  max-height: 25rem;
-  box-shadow: 0 0 15px rgba(0, 0, 0, 0.2);
+.ZoomableImage {
+  box-shadow: 0 0 15px rgba(24, 96, 141, 0.2);
   margin: 15px 0 0;
+  object-fit: contain;
+  max-height: 350px;
+  max-width: 100%;
   cursor: zoom-in;
 }
 
-.GrowImage:focus,
-.GrowImage:active {
+.ImageOverlayBackdrop {
   cursor: zoom-out;
-  max-height: unset;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  backdrop-filter: blur(5px);
```

Personally I like the background dimmed and less blurred, like this.
```suggestion
  backdrop-filter: brightness(60%) blur(2px);
```